proposals/csharp-10.0/improved-interpolated-strings.md

```diff
@@ -452,6 +452,46 @@ rented array inside it, and if one of the interpolation holes throws an exceptio
 _Answer_: No. handlers can be assigned to locals (such as `MyHandler handler = $"{MyCode()};`), and the lifetime of such handlers is unclear. Unlike foreach enumerators, where the lifetime
 is obvious and no user-defined local is created for the enumerator.
 
+### Impact on nullable reference types
+
+To minimize complexity of the implementation, we have a few limitations on how we perform nullable analysis on interpolated string handler constructors used as arguments to a method or indexer.
+In particular, we do not flow information from the constructor back through to the original slots of parameters or arguments from the original context, and we do not use constructor parameter
+types to inform generic type inference for type parameters in the containing method. An example of where this can have an impact is:
+
+```cs
+string s = """";
```

Comment:
Was it intended to have doubled quote marks here and below? This looks like it might have been left over from pasting in a test.